libsolidity/formal/ModelChecker.cpp

```diff
@@ -135,9 +141,12 @@ vector<string> ModelChecker::unhandledQueries()
 	return m_bmc.unhandledQueries() + m_chc.unhandledQueries();
 }
 
-solidity::smtutil::SMTSolverChoice ModelChecker::availableSolvers()
+SMTSolverChoice ModelChecker::availableSolvers()
 {
 	smtutil::SMTSolverChoice available = smtutil::SMTSolverChoice::SMTLIB2();
+#if defined(__linux) || defined(__APPLE__)
```

Comment:
Don't we also need one for windows here? What about the `EMSCRIPTEN` flag that Chris mentioned - does it exist?

Reply:
Looks like we at least have it as a CMake option:
https://github.com/ethereum/solidity/blob/3109ce2dbcb07606df0b177b8e0378f6d6b80835/CMakeLists.txt#L139-L141

Reply:
we don't really support using the smtchecker on windows anyway without compiling the compiler yourself, and we don't test on windows, so left it out for now. That's also why I didn't try the EMSCRIPTEN one

Reply:
ok. I guess it's fine then as long as the message says that eld is not supported on this platform (rather than emscripten specifically).